pkg/app/app_test.go

```diff
@@ -2596,6 +2598,10 @@ func (helm *mockHelmExec) IsVersionAtLeast(versionStr string) bool {
 	return false
 }
 
+func (helm *mockHelmExec) ShowChart(chartPath string) (chart.Metadata, error) {
+	return chart.Metadata{}, errors.New("fake test error")
```

Comment:
why return chart.Metadata{}, errors.New("fake test error")?
we can do the same logic with other func for `*mockHelmExec`

Reply:
this is to retain the existing tests which rely on the failure of this method.

Reply:
ok. got it.

Reply:
I updated the error message to make it clearer though.